build/embed.bzl

```diff
@@ -66,6 +66,21 @@ def _clean_path(directory):
      return _clean_string(parts[1])
    return _clean_string(parts[0])
 
+def _strip_extensions(s, extensions):
+   """Remove all the specified extensions from the supplied string.
+   
+   Args:
+     s: string, generally a filename.
+     extensions: list of strings, extensions to remove.
```

Comment:
nit: You might want to mention that the order of extensions is important. If I'm reading this correctly you must supply them in reverse order relative to 's'

Reply:
good suggestion, removed.